workflows/utilities/data_import/wf_fetch_srr_accession.wdl

```diff
@@ -0,0 +1,26 @@
+version 1.0
+
+import "../../../tasks/utilities/data_handling/task_fetch_srr_accession.wdl" as srr_task
+import "../../../tasks/task_versioning.wdl" as versioning_task
+
+workflow fetch_srr {
```

Comment:
can you call this fetch_srr_accession?

if you're having namespace conflicts, you could change line 16 to something like `call srr_task.fetch_srr_accession as fetch_srr` or something? I'd rather the workflow name be the full name and the task name be truncated if necessary